action.yml

```diff
@@ -20,15 +24,17 @@ runs:
   using: composite
   steps:
     - name: Check out the parent repository with student submission data
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
     - name: Extract the student submission repo from URL
-      uses: actions/github-script@v6
+      uses: actions/github-script@v7
       id: get-repo-name
       continue-on-error: true
       with:
         result-encoding: string
         script: |
-          const submission = require('./submission.json');
+          const submissionJsonPathOverride = "${{ inputs.submissionJsonPath }}";
+          const submissionJsonPath = submissionJsonPathOverride === "" ? "submission.json" : submissionJsonPathOverride;
+          const submission = require(process.env.GITHUB_WORKSPACE + "/" + submissionJsonPath);
           const gitURL =  submission.checklist[0].result;
```

Comment:
I think we should document this bit; I was of the assumption that position is inconsequential. (In plagiarism app we find the `url` from the `checklist`)